### PR TITLE
script to add project metadata

### PR DESCRIPTION
**Pull request by kshitij12345**
Script to add project metadata

### CODE REVIEW

File: src/project_metadata.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import plotly.express as px
+
+project = neptune.init_project()
+
+data = '../data/covid_and_healthy_spectra.csv'
+
+df = pd.read_csv(data)
+
+# Upload data
+project["data/files"].upload(data)
```
Comment (SiddhantSadangi) [inline src/project_metadata.py]: This is a large file and rendering it slows down the browser. Use `track_files()` instead

File: src/project_metadata.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+# Histogram of classes present in the data
+project["data/class_balance"] = neptune.types.File.as_html(px.histogram(df.diagnostic))
+
+# Upload samples
+samples = df.sample(20)
+project["data/samples"].upload(File.from_content(samples.to_html(), extension="html"))
```
Comment (SiddhantSadangi) [inline src/project_metadata.py]: Upload sample as csv.
Refer to https://app.neptune.ai/o/showcase/org/project-text-classification/metadata?path=data%2F

**Review by SiddhantSadangi — CHANGES_REQUESTED** [review]

File: src/project_metadata.py (hunk 1 above)
Comment (kshitij12345) [reply src/project_metadata.py]: For future reference, what is the definition of large file? Cause this one is around 6 MB.

Comment (kshitij12345) [reply src/project_metadata.py]: Addressed.

File: src/project_metadata.py (hunk 2 above)
Comment (kshitij12345) [reply src/project_metadata.py]: Addressed.

**Review by SiddhantSadangi — APPROVED** [review]

**Pull request merged**